mos-platform/eater/link.ld

```diff
@@ -0,0 +1,49 @@
+/* Linker script for Ben Eater's 6502 breadboard computer. */
+
+/* Provide imaginary (zero page) registers. */
+__rc0 = 0xe0;
+INCLUDE imag-regs.ld
+ASSERT(__rc31 == 0xff, "Inconsistent zero page map.")
+
+MEMORY {
+  zp : ORIGIN = 0x00, LENGTH = 0xe0
+  ram : ORIGIN = 0x200, LENGTH = 0x4000 - 0x200
+  rom : ORIGIN = 0x8000, LENGTH = 0x7ffa
+}
+
+/* Provide default IRQ and NMI handlers if the program doesn't have them. */
+PROVIDE(irq = _irq_default);
+PROVIDE(nmi = _irq_default);
+
+SECTIONS {
+  .text : {
+       INCLUDE text-sections.ld
+  } >rom AT>rom
+  .rodata : { INCLUDE rodata-sections.ld } >rom AT>rom
+  .data : { INCLUDE data-sections.ld } >ram AT>rom
+  __data_load_start = LOADADDR(.data) - LOADADDR(.rodata) + ADDR(.rodata);
+  __data_size = SIZEOF(.data);
+  .zp.data : { INCLUDE zp-data-sections.ld } >zp AT>rom
+  __zp_data_load_start = LOADADDR(.zp.data) - LOADADDR(.rodata) + ADDR(.rodata);
+  __zp_data_size = SIZEOF(.zp.data);
+  .data_initializers ADDR(.rodata) + SIZEOF(.rodata) + __data_size +
```

Comment:
The `.data_initializers` section was a symptom of the older split LMA/VMA approach that I'd used on banked ROM targets, but it ended up being unnecessary. I had reworked the NES targets so this isn't needed, but had forgotten to go back and re-do `atari-stdcart`; unlucky.

In any case, it's unnecessary here, since the ROM has both its LMA and VMA in the same memory region: `rom`. Accordingly, you can just get rid of `.data_initializers`.